airflow/www/static/js/dag/details/taskInstance/Details.tsx

```diff
@@ -24,7 +24,8 @@ import {
   Table,
   Tbody,
   Tr,
-  Td, Heading, Thead,
+  Td,
+  Thead,
```

Comment:
I think this file has been updated. You may want to rebase.

Reply:
Yes there is a big rebase coming in a bit I think xD